src/vnm_mad/remotes/vxlan/vxlan.rb

```diff
@@ -92,23 +94,26 @@ def list_interface_vlan(name)
     end
 
     def get_interface_first_ip(name)
-        text = %x(#{command(:ip_unpriv)} addr show dev #{name})
-        return nil if $?.exitstatus != 0
+        text = `#{command(:ip_unpriv)} addr show dev #{name}`
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Detected a dangerous subshell execution vulnerability where user input is directly used in a shell command without proper sanitisation. This could allow an attacker to inject and execute arbitrary system commands, potentially leading to system compromise or data breach. To remediate, avoid using shell commands with user input when possible, or implement strict input validation and use shell escaping functions like Shellwords.escape(). If shell execution is necessary, consider using a allow list of allowed commands and arguments. Link to more info : https://guides.rubyonrails.org/security.html